samples/net/openthread/border_router/prj.conf

```diff
@@ -0,0 +1,101 @@
+CONFIG_EARLY_CONSOLE=y
+# Generic networking options
+CONFIG_NETWORKING=y
+CONFIG_NET_UDP=y
+CONFIG_NET_TCP=y
+CONFIG_NET_IPV6=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_DHCPV4=y
+CONFIG_NET_MGMT_EVENT=y
+CONFIG_NET_CONNECTION_MANAGER=y
+CONFIG_NET_NATIVE=y
+CONFIG_NET_ROUTE_MCAST=y
+CONFIG_NET_MAX_MCAST_ROUTES=10
+CONFIG_NET_IPV6_NBR_CACHE=y
+CONFIG_OPENTHREAD_ZEPHYR_BORDER_ROUTER=y
+
+
+# printing of scan results puts pressure on queues in new locking
+# design in net_mgmt. So, use a higher timeout for a crowded
+# environment.
+CONFIG_NET_MGMT_EVENT_QUEUE_TIMEOUT=5000
+CONFIG_NET_MGMT_EVENT_QUEUE_SIZE=16
+
+# Generic OT options
+CONFIG_NET_L2_OPENTHREAD=y
+CONFIG_OPENTHREAD_SHELL=y
+CONFIG_OPENTHREAD_DEBUG=n
+CONFIG_OPENTHREAD_L2_DEBUG=n
+CONFIG_OPENTHREAD_THREAD_VERSION_1_3=y
+CONFIG_OPENTHREAD_L2_LOG_LEVEL_INF=n
+
+# Kernel options
+CONFIG_MAIN_STACK_SIZE=32768
+CONFIG_SHELL_STACK_SIZE=8192
+CONFIG_OPENTHREAD_THREAD_STACK_SIZE=20480
+CONFIG_LOG_PROCESS_THREAD_STACK_SIZE=16384
+
+CONFIG_NET_TX_STACK_SIZE=2048
+CONFIG_NET_RX_STACK_SIZE=2048
+CONFIG_ENTROPY_GENERATOR=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_INIT_STACKS=y
+
+# Logging
+CONFIG_NET_LOG=n
+CONFIG_LOG=y
+CONFIG_LOG_PROCESS_THREAD=y
+CONFIG_NET_STATISTICS=y
+CONFIG_NET_STATISTICS_PERIODIC_OUTPUT=n
+CONFIG_PRINTK=y
+
+# Network buffers
+CONFIG_NET_PKT_RX_COUNT=16
+CONFIG_NET_PKT_TX_COUNT=16
+CONFIG_NET_BUF_RX_COUNT=64
+CONFIG_NET_BUF_TX_COUNT=64
+CONFIG_NET_CONTEXT_NET_PKT_POOL=y
+
+# IP address options
+CONFIG_NET_IF_UNICAST_IPV6_ADDR_COUNT=6
+CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=6
+CONFIG_NET_MAX_CONTEXTS=10
+CONFIG_NET_IF_MAX_IPV6_COUNT=2
+
+
+# Number of socket descriptors might need adjusting
+# if there are more than 1 handlers defined.
+CONFIG_ZVFS_OPEN_MAX=15
+
+# OT Border Router flags
+# OPENTHREAD_CONFIG_MULTICAST_DNS_PUBLIC_API_ENABLE=y
```

Comment:
I'd prefer to remove commented out options, unelss explictly documented in the comment why is it present.